src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/PackageLibs.targets

```diff
@@ -140,6 +140,12 @@
       <FilesToPackage Condition="'%(FilesToPackage.SubFolder)' != ''">
         <TargetPath>%(TargetPath)%(FilesToPackage.SubFolder)</TargetPath>
       </FilesToPackage>
+
+      <!-- Include source files for symbol packages. -->
+      <FilesToPackage Include="@(Compile->'%(FullPath)')" Condition="$([System.String]::new('%(FullPath)').StartsWith('$(ProjectDir)'))">
+        <TargetPath>src\$([System.String]::new('%(FullPath)').Replace('$(ProjectDir)', ''))</TargetPath>
```

Comment:
Have you looked at [$([MSBuild]::MakeRelative())](https://msdn.microsoft.com/en-us/library/dd633440.aspx#Anchor_6)?